From: =?utf-8?q?Guido=20G=C3=BCnther?= <agx@sigxcpu.org>
Date: Thu, 2 Jul 2009 18:08:16 +0200
Subject: [PATCH] don't run configure

leave this to debian/rules so we pass the right args
---
 autogen.sh |    1 -
 1 files changed, 0 insertions(+), 1 deletions(-)

diff --git a/autogen.sh b/autogen.sh
index a1d7e27..731e44d 100755
--- a/autogen.sh
+++ b/autogen.sh
@@ -28,4 +28,3 @@ autoreconf -i
 pushd daemon
 autoreconf -i
 popd
-./configure "$@"
-- 
